mapie/utils.py

```diff
@@ -271,15 +271,16 @@ def check_n_features_in(
     if cv == "prefit" and hasattr(estimator, "n_features_in_"):
         if cast(Any, estimator).n_features_in_ != n_features_in:
             raise ValueError(
-                "Invalid mismatch between "
+                "Invalid mismatch between ",
                 "X.shape and estimator.n_features_in_."
             )
     return n_features_in
 
 
 def check_alpha_and_n_samples(
```

Comment:
I believe this is the correct indetation?

def check_alpha_and_n_samples(
        alphas: Union[Iterable[float], float],
        n: int
) -> None: